app/src/main/java/com/github/mobile/ui/ViewPager.java

```diff
@@ -61,7 +59,7 @@ public boolean setItem(final int item) {
     /**
      * Set current item, invoke the listener if changes, and return whether the
      * item changed
-     * <p>
+     * <p/>
```

Comment:
Haven't noticed it. It was from the IDE by AutoFormatting :)
Will change it back soon...